src/components/Page/PageHero.jsx

```diff
@@ -122,23 +123,31 @@ const PageHero = ({
   leftContent,
   rightContent,
   switcher,
-}) => (
-  <StyledPageHero className={className}>
-    {crumb || (
-      <Fragment>
-        {leftContent ? <StyledLeftContent>{leftContent}</StyledLeftContent> : null}
-        <StyledTitle>
-          <PageTitle section={section} title={title} />
-          {switcher && switcher.options.length ? <PageSwitcher switcher={switcher} /> : null}
-          <StyledPageHeroWrap>
-            {blurb ? <StyledPageBlurb>{blurb}</StyledPageBlurb> : null}
-          </StyledPageHeroWrap>
-        </StyledTitle>
-        {rightContent ? <StyledRightContent>{rightContent}</StyledRightContent> : null}
-      </Fragment>
-    )}
-  </StyledPageHero>
-);
+}) => {
+  if (__DEV__) {
+    warning(
+      false,
+      'PageHero component has been deprecated and will be removed in the next release of `carbon-addons-iot-react`. \n Refactor to use Hero component instead.'
```

Comment:
deprecation should refer to PageTitleBar instead of Hero, correct?

Reply:
oo yes, good catch! This should be `PageTitleBar` instead of `Hero`